Cx/NfcCxSE.cpp

```diff
@@ -3138,6 +3183,31 @@ Return Value:
                           RtngTableEntry.LstnModeRtngValue.tAidBasedRtngValue.bAidSize);
             status = IsEqualGUID(pRoutingTable->TableEntries[i].AidRoutingInfo.guidSecureElementId, GUID_NULL) ? STATUS_INTERNAL_ERROR : STATUS_SUCCESS;
             break;
+        case phNfc_LstnModeRtngSystemCodeBased:
+            pRoutingTable->TableEntries[i].eRoutingType = RoutingTypeSystemCode;
+            pRoutingTable->TableEntries[i].SystemCodeRoutingInfo.guidSecureElementId = NfcCxSEInterfaceGetSecureElementId(RFInterface, RtngTableEntry.hSecureElement);
+            pRoutingTable->TableEntries[i].SystemCodeRoutingInfo.cbSystemCode = RtngTableEntry.LstnModeRtngValue.tSystemCodeBasedRtngValue.bSystemCodeSize;
+
+            _Analysis_assume_(MAXIMUM_SYSTEM_CODE_LENGTH >= RtngTableEntry.LstnModeRtngValue.tSystemCodeRtngValue.bAidSize);
```

Comment:
RtngTableEntry.LstnModeRtngValue.tSystemCodeRtngValue.**bAidSize** should be replaced with .**bSystemCodeSize**

Reply:
ok